src/screens/app/Components/CommentsTouchBox/CommentsTouchBox.js

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable react-native/no-inline-styles */
```

Comment:
Ensure this file doesn't conflict with what's on master. Otherwise remove commit.